src/sync/app.cpp

```diff
@@ -927,6 +927,10 @@ void App::call_function(std::shared_ptr<SyncUser> user,
         if (auto error = check_for_errors(response)) {
             return completion_block(error, util::none);
         }
+        
+        if (response.body == "null") {
```

Comment:
I'm not sure this correct. "null" is technically valid BSON.

Reply:
I've removed this check, I had some weird runtime exceptions where a "null" response body was not parsing correctly but it looks to be all good now